packages/cli/src/handlers/dbt/run.ts

```diff
@@ -54,8 +60,11 @@ export const dbtRunHandler = async (
         });
         throw new ParseError(`Failed to run dbt:\n  ${msg}`);
     }
-    await generateHandler({
-        ...options,
-        excludeMeta: options.excludeMeta,
-    });
+
```

Comment:
perhaps this would be useful?

```suggestion
if (options.assumeYes && options.assumeNo) {
    throw new Error("Cannot use both --assume-yes and --assume-no flags");
}
```

Reply:
Added in [chore: throw when both options are used](https://github.com/lightdash/lightdash/pull/13543/commits/176a82c7c063419204f89056fa147bae94b3d764)